tools/map2header.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3
```

Comment:
might be problematic if there are breaking changes in the future python3.x versions